internal/services/containers/kubernetes_cluster_network_resource_test.go

```diff
@@ -258,6 +258,66 @@ func TestAccKubernetesCluster_advancedNetworkingAzureCiliumPolicyUpdate(t *testi
 	})
 }
 
+func TestAccKubernetesCluster_advancedNetworkingAzureAzurePolicyUpdate(t *testing.T) {
```

Comment:
A test that validates omitting `network_policy` after it has been set is misleading in this instance because the property is `Computed`. 

The tests should validate that we can update an existing cluster with no network policy to having a network policy of `azure` and or `calico`.